tests/src/Traits/OgTrait.php

```diff
@@ -70,4 +72,33 @@ protected function convertOgRoleNamesToIds(array $roles, EntityInterface $entity
     return $roles;
   }
 
+  /**
+   * Asserts the ownership of a user to a given group.
+   *
+   * An ownership is defined as having a specific collection of roles in that
+   * group.
+   *
+   * @param AccountInterface $user
+   *    The user to be checked.
+   * @param RdfInterface $group
+   *    The group entity. In this project, only rdf entities are groups.
+   * @param array $roles
+   *    An array of roles to be checked. Roles must be passed as simple names
+   *    and not as full IDs. Names will be converted accordingly to IDs.
+   *
+   * @throws \Exception
+   *    Throws exception when the user is not a member or is not an owner.
+   */
+  protected function assertOgGroupOwnership(AccountInterface $user, RdfInterface $group, $roles) {
+    $membership = Og::getMembership($user, $group);
+    if (empty($membership)) {
+      throw new \Exception("User {$user->name} is no member of the {$group->label()} group.");
```

Comment:
"is not a member" and it should be "{$user->getAccountName()}" or an exception is thrown.